Custom node development setup error

**Describe the bug**
Unable to set up the dev environment as described in the docs:
https://docs.n8n.io/nodes/creating-nodes/create-node.html#prerequisites

Error in Step:
https://docs.n8n.io/nodes/creating-nodes/create-node.html#adding-the-node-to-editor-ui
when trying to run ```lerna bootstrap --hoist```

**To Reproduce**
Steps to reproduce the behavior:
1. Install Git & latest Nodejs
2. Git clone latest n8n repo
3. cd into n8n and run ```lerna bootstrap --hoist```
4. See error:
```
$ sudo lerna bootstrap --hoist
info cli using local version of lerna
lerna notice cli v3.22.1
lerna info versioning independent
lerna info Bootstrapping 6 packages
lerna WARN EHOIST_PKG_VERSION "n8n" package depends on @types/node@14.0.27, which differs from the hoisted @types/node@^14.14.40.
lerna WARN EHOIST_PKG_VERSION "n8n-editor-ui" package depends on uuid@^8.1.0, which differs from the hoisted uuid@^3.4.0.
lerna WARN EHOIST_PKG_VERSION "n8n-node-dev" package depends on @types/tmp@^0.1.0, which differs from the hoisted @types/tmp@^0.2.0.
lerna WARN EHOIST_PKG_VERSION "n8n-node-dev" package depends on tmp-promise@^2.0.2, which differs from the hoisted tmp-promise@^3.0.2.
lerna info Installing external dependencies
lerna info hoist Installing hoisted dependencies into root
lerna info hoist Pruning hoisted dependencies
lerna info hoist Finished pruning hoisted dependencies
lerna ERR! npm install exited 1 in 'n8n'
lerna ERR! npm install stderr:
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: n8n@undefined
npm ERR! Found: eslint@6.8.0
npm ERR! node_modules/eslint
npm ERR!   eslint@"^6.8.0" from the root project
npm ERR!   peer eslint@">=6.2.2" from @vue/eslint-config-standard@5.1.2
npm ERR!   node_modules/@vue/eslint-config-standard
npm ERR!     @vue/eslint-config-standard@"^5.0.1" from the root project
npm ERR!   2 more (eslint-plugin-import, eslint-plugin-node)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^7.0.0" from eslint-plugin-promise@5.1.0
npm ERR! node_modules/eslint-plugin-promise
npm ERR!   peer eslint-plugin-promise@">= 4.2.1" from @vue/eslint-config-standard@5.1.2
npm ERR!   node_modules/@vue/eslint-config-standard
npm ERR!     @vue/eslint-config-standard@"^5.0.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /root/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-04-22T09_56_27_250Z-debug.log

lerna ERR! npm install exited 1 in 'n8n'

```

**Expected behavior**
No error, so that the next steps could then be executed:
```
npm run build
npm run dev
```

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04 as well as Windows 10 (10.0.19042)
 - n8n Version 0.116.0
 - Node.js Version 14.16.1

--- a/.npmrc
+++ b/.npmrc
@@ -1,0 +1,1 @@
+legacy-peer-deps=true

